src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -1569,15 +1569,13 @@ public void DisableSearchTextBox()
         }
 
         /// <summary>
-        /// Clear after closing down
+        /// Only suspend the elements associated with the view
         /// </summary>
-        internal void Close()
+        internal void PackageManagerViewClose()
```

Comment:
is this a handler? Maybe it should be called OnPackageManagerViewClose?

Reply:
It is actively called from inside the View when the View is about to close. I don't know if the VM has handle on the View to subscribe to an event, if so I can flip it around, as in subscribe to the View Closing or Closed event.

Reply:
I checked, and the view model does not reference the view. Would that be an acceptable solution?